src/main/frontend/components/reference.cljs

```diff
@@ -97,6 +97,5 @@
             (if @n-ref
               (str @n-ref " Unlinked References")
               "Unlinked References")]
-           (fn []
-             (unlinked-references-aux page-name n-ref))
+           (unlinked-references-aux page-name n-ref)
```

Comment:
It need to be a function to be lazy loaded.

Reply:
Got it.  I am curious about passing a function as a parameter, the browser will complaint this:
`Functions are not valid as a React child. This may happen if you return a Component instead of <Component /> from render. Or maybe you meant to call this function rather than return it.`

Reply:
let's change the frontend.ui/foldable to return a nil if it's a `fn` and collapsed? is `true`, something like thisL
```
(cond
 (and (fn? content) (not @collapsed?))
 (content)

 (fn? content)
 nil

 :else
  content)
```

Reply:
Excellent solution, I think it will work.